content/docs/csidriver/upgradation/drivers/powerflex.md

```diff
@@ -10,9 +10,9 @@ Description: Upgrade PowerFlex CSI driver
 
 You can upgrade the CSI Driver for Dell PowerFlex using Helm or Dell CSI Operator.
 
-## Update Driver from v2.4 to v2.5 using Helm
+## Update Driver from v2.5 to v2.6 using Helm
 **Steps**
-1. Run `git clone -b v2.5.0 https://github.com/dell/csi-powerflex.git` to clone the git repository and get the v2.3.0 driver.
+1. Run `git clone -b v2.6.0 https://github.com/dell/csi-powerflex.git` to clone the git repository and get the v2.3.0 driver.
```

Comment:
Should
"and get the v2.3.0 driver" be changed to 2.6.0?

Reply:
updated